app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -161,11 +170,11 @@ class ConversationListViewModel @Inject constructor(
                         ),
                         conversationInfo = ConversationInfo(
                             name = otherUser.name.orEmpty(),
-                            membership = mapUserType(details.userType),
-                            isLegalHold = false
+                            membership = mapUserType(details.userType)
                         ),
                         conversationId = conversation.id,
-                        mutedStatus = conversation.mutedStatus
+                        mutedStatus = conversation.mutedStatus,
+                        isLegalHold = details.legalHoldStatus == LegalHoldStatus.ENABLED
```

Comment:
Can you extract details.legalHoldStatus == LegalHoldStatus.ENABLED to a separate function nstead of checking it everywhere ?